config/common_base

```diff
@@ -810,6 +810,7 @@ CONFIG_RTE_LIBRTE_PMD_NTB_RAWDEV=y
 #
 CONFIG_RTE_LIBRTE_REGEXDEV=y
 CONFIG_RTE_LIBRTE_REGEXDEV_DEBUG=n
+CONFIG_RTE_MAX_REGEXDEV_PORTS=32
```

Comment:
@orikam we do not have conept of regex ports.

we have only regex devices.
So lets just say devices here.
CONFIG_RTE_REGEX_MAX_DEVS=32

Reply:
OK will change to dev